tests/src/common/swap.rs

```diff
@@ -17,6 +18,28 @@ use super::{
     ADMIN,
 };
 
+#[repr(transparent)]
+#[derive(Debug, Eq)]
+pub struct DexDenom<'r>(&'r str);
```

Comment:
could you outline the motivation behind your decision to define it?

Reply:
So it is clear a DEX symbol is passed down and not a bank one. The whole motivation was to avoid ambiguity.